repo/cfn/inc.mk

```diff
@@ -116,6 +124,11 @@ $(CFN_JSON_FILES): %.cfn.json: %/index.js %-setup %.cfn.json/lint ## Generate st
 %.cfn.json/rm: %-setup ## Remove stack.
 	$(ECHO_DO) "Removing $(STACK_NAME) stack..."
 	$(call $(STACK_STEM)-pre-rm)
+	$(ECHO_DO) "Removing $(STACK_NAME) stack policy (allowing all changes)..."
```

Comment:
maybe execute this before line 125? so that whatever in `pre-rm` can actually be done?